### DISCUSSION TITLE
System service?

### DISCUSSION THREAD

**Discussion by darlopvil** [FlareSolverr/FlareSolverr#923] [Q&A] [answered]
Hello guys,

New here, first time running this program and i was wondering if it can start automatically as a Windows system service. All I've seen is this little section in the readme: https://github.com/FlareSolverr/FlareSolverr#systemd-service
I don't have a clue how that can help me. What is `flaresolverr.service`? It refers it as a file but didn't fine anything in the downloaded folder.

Is there a way to make flaresolverr start automatically on the system startup?

**Comment by ilike2burnthing** [comment, marked as answer]
https://github.com/FlareSolverr/FlareSolverr/issues/70

**Comment by Rid** [comment]
For anyone looking. The flareserverr.service can be found here: https://github.com/FlareSolverr/FlareSolverr/blob/345628e3e40fddf337ea74f7ea6ca318edf8a054/flaresolverr.service
